test/unit_tests/cvode/CXX_serial/CMakeLists.txt

```diff
@@ -15,19 +15,22 @@
 # ---------------------------------------------------------------
 
 # List of test tuples of the form "name\;args"
-set(unit_tests
-  "cv_test_kpr.cpp\;"
-  "cv_test_kpr.cpp\;--eta_min_fx 1.0 --eta_max_fx 2.0"
-  "cv_test_kpr.cpp\;--eta_max_fs 2"
-  "cv_test_kpr.cpp\;--eta_min_es 2 --small_nst 5"
-  "cv_test_kpr.cpp\;--eta_min_gs 2"
-  "cv_test_kpr.cpp\;--eta_min_fx 1.0 --eta_min 0.5"
-  "cv_test_kpr.cpp\;--eta_min_ef 0.5"
-  "cv_test_kpr.cpp\;--eta_max_ef 0.1 --small_nef 1"
-  "cv_test_kpr.cpp\;--eta_cf 0.5"
-  "cv_test_kpr.cpp\;--dgmax_lsetup 0.0"
-  "cv_test_kpr.cpp\;--dgmax_jbad 1.0"
+if (SUNDIALS_PRECISION MATCHES "DOUBLE")
+  set(unit_tests
+    "cv_test_kpr.cpp\;"
+    "cv_test_kpr.cpp\;--eta_min_fx 1.0 --eta_max_fx 2.0"
+    "cv_test_kpr.cpp\;--eta_max_fs 2"
+    "cv_test_kpr.cpp\;--eta_min_es 2 --small_nst 5"
+    "cv_test_kpr.cpp\;--eta_min_gs 2"
+    "cv_test_kpr.cpp\;--eta_min_fx 1.0 --eta_min 0.5"
+    "cv_test_kpr.cpp\;--eta_min_ef 0.5"
+    "cv_test_kpr.cpp\;--eta_max_ef 0.1 --small_nef 1"
+    "cv_test_kpr.cpp\;--eta_cf 0.5"
+    "cv_test_kpr.cpp\;--dgmax_lsetup 0.0"
+    "cv_test_kpr.cpp\;--dgmax_jbad 1.0"
+    "cv_test_getjac.cpp\;"
```

Comment:
If we update the `*_test_getjac.cpp` routines as I recommended, then these would not need to require `SUNDIALS_PRECISION MATCHES "DOUBLE"`